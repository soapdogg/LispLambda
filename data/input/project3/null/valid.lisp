(NULL NIL)
(NULL (NULL NIL))
(ATOM (NULL (INT (CONS 34 54))))
(NULL (QUOTE (A)))
(NULL (EQ 2 (PLUS 1 1)))