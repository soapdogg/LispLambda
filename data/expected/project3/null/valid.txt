T
NIL
T
NIL
NIL
